.gitignore

```diff
@@ -1,3 +1,3 @@
-bin/
-gopath/
-*.swp
+/target/
+**/*.rs.bk
+Cargo.lock
```

Comment:
Why aren't we checking in `Cargo.lock`?

Reply:
we can. iirc this is just how cargo set it up when I started the project. maybe that's one of the `cargo new --bin` vs `cargo new --lib` differences.

Reply:
It is; `bin` leaves it unignored.

Isn't this a `bin` not a `lib` relaly though?

Reply:
it's a little of both. it's the command line utility but also a library consumed by `coreos-metadata`. I can include it.